readme.md

```diff
@@ -28,6 +28,7 @@ Most of the time you will be interacting with arduino with a client library on t
   * [http://funnel.cc]
 * clojure
   * [https://github.com/nakkaya/clodiuno]
+  * [https://github.com/peterschwarz/clj-firmata]
```

Comment:
It's strange that the 2 diffs in this file are showing up because these two links are already in master. Make sure you're fork is in sync with master. I know a while back I asked you to submit against the `dev` branch but I have since decided not to use `dev` anymore and instead am not asking people to submit pull request against master (sorry about any confusion that may have caused).